README.md

```diff
@@ -287,6 +321,8 @@ import net.conjur.api.Conjur;
 
 // Configured using environment variables
 Conjur conjur = new Conjur();
+// or using custom SSLContext setup as conjurSslContext variable
+Conjur conjur = new Conjur(conjurSslContext);
```

Comment:
I see mentions of both `SSLContext` and `SslContext`. Any reason for this or can we standardise on `SSLContext`?

Reply:
👍 I'll swap them over - this is just in the readme and the examples are using that syntax.